src/dialogs/more-info-dialog.html

```diff
@@ -16,6 +16,11 @@
     <style>
       paper-dialog {
         font-size: 14px;
+        width: 450px;
+      }
+
+      paper-dialog[data-domain=camera] {
+        min-width: auto;
```

Comment:
No…

The min-width should stay 450 pixels and the width should be set to auto. Now the camera images will still be capped to 450 pixels even though they might have more to show.